Documentation/ceph-upgrade.md

```diff
@@ -312,12 +312,26 @@ kubectl apply -f common.yaml -f crds.yaml
 
 #### **Updates for optional resources**
 
-If you have [Prometheus monitoring](ceph-monitoring.md) enabled, follow the
-step to upgrade the Prometheus RBAC resources as well.
-
-```sh
-kubectl apply -f deploy/examples/monitoring/rbac.yaml
-```
+* [Prometheus monitoring](ceph-monitoring.md): upgrade Prometheus RBAC resources with this command.
+  ```sh
+  kubectl apply -f deploy/examples/monitoring/rbac.yaml
+  ```
+
+* [Admission controller](admission-controller-usage.md): The admission controller has been
```

Comment:
Should this be part of step 3 where the operator is updated? This is inside step 1 where the RBAC and CRDs are updated.

Reply:
I think it's fine to do it here. It is an optional resource, and so there is already a section for it. A user **could** get mixed up an accidentally skip step 2, but that step is also optional of a sort, so I'm not terribly worried about it breaking things.